app/config/config.yml

```diff
@@ -1,5 +1,6 @@
 imports:
     - { resource: parameters.yml }
+    - { resource: heroku.php, ignore_errors: true }
```

Comment:
I see.
The point I don't like here is that this require resources should grow with all external adaptations (in that case, Heroku...). Is this the good way?